libs/designer/src/lib/core/parsers/BJSWorkflow/BJSDeserializer.ts

```diff
@@ -51,6 +52,16 @@ export const Deserialize = (
       ...addTriggerInstanceMetaData(runInstance),
     };
     allActionNames.push(tID);
+  } else {
+    // Workflow has no trigger, create a placeholder trigger node to reference during initialization
+    const tID = constants.NODE.TYPE.PLACEHOLDER_TRIGGER;
+    triggerNode = createWorkflowNode(tID, WORKFLOW_NODE_TYPES.PLACEHOLDER_NODE);
```

Comment:
I thought we already had that or maybe it was only for totally empty designer which has no actions as well.

Reply:
Completely empty designer is handled fine, this is just when there's other actions that rely on there being a trigger to reference in the graph